samples/net/vlan/Kconfig

```diff
@@ -53,4 +53,12 @@ config NET_SAMPLE_IFACE3_VLAN_TAG
 	  Set VLAN (virtual LAN) tag (id) that is used in the sample
```

Comment:
and the commit subject of the second one to `samples: net: vlan: Add IPv6 prefix config to each vlan-iface`